Formula/wiki.rb

```diff
@@ -13,13 +13,13 @@ class Wiki < Formula
     sha256 "271812a50a0455e1947006624f0442f084c69ab387cc858d41be26a4cce195f0" => :mavericks
   end
 
+  depends_on "go" => :build
+
   go_resource "github.com/mattn/go-colorable" do
     url "https://github.com/mattn/go-colorable.git",
       :revision => "40e4aedc8fabf8c23e040057540867186712faa5"
   end
 
-  depends_on "go" => :build
-
   conflicts_with "osxutils", :because => "both install `wiki` binaries"
```

Comment:
Mind moving this up with it? `depends_on` and `conflicts_with` are supposed to be fairly close to each other as it was.